dist/output.css

```diff
@@ -1,5 +1,5 @@
 /*
-! tailwindcss v3.2.7 | MIT License | https://tailwindcss.com
+! tailwindcss v3.2.6 | MIT License | https://tailwindcss.com
```

Comment:
@fplord pls switch to pnpm so that the lockfile is respected and we have consistent versions